src/RoslynPad.Runtime/Runtime/ProgressReport.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace RoslynPad.Runtime
+{
+    /// <summary>
+    /// Reports progress to RoslynPad GUI
+    /// </summary>
+    public static class ProgressReport
```

Comment:
Remove this class. I'd like a single entry point for RP APIs, and that's `Helpers`.

Reply:
removed